src/include/storage/copier/column_chunk.h

```diff
@@ -93,8 +93,11 @@ class ColumnChunk {
     template<typename T>
     void templateCopyArrowArray(
         arrow::Array* array, common::offset_t startPosInChunk, uint32_t numValuesToAppend);
-    // TODO(Guodong/Ziyi): The conversion from string to values should be handled inside ReadFile.
     template<typename T>
+    void templateCopyStringArrowArray(
+        arrow::Array* array, common::offset_t startPosInChunk, uint32_t numValuesToAppend);
+    // TODO(Guodong/Ziyi): The conversion from string to values should be handled inside ReadFile.
+    template<typename KU_TYPE, typename ARROW_TYPE>
```

Comment:
Can you add a comment here to explain that `ARROW_TYPE` can be STRING or LARGE_STRING? It's not self-evident that why we need another typename as the arrow array is guaranteed to be string.